internal/restic/node_xattr.go

```diff
@@ -16,7 +16,8 @@ import (
 // Getxattr retrieves extended attribute data associated with path.
 func Getxattr(path, name string) ([]byte, error) {
 	b, e := xattr.Get(path, name)
-	if err, ok := e.(*xattr.Error); ok && err.Err == syscall.ENOTSUP {
+	if err, ok := e.(*xattr.Error); ok &&
+		(err.Err == syscall.ENOTSUP || err.Err == syscall.ENODATA) {
```

Comment:
Apparently FreeBSD (see Travis failures) does not have `syscall.ENODATA`. As the error has only been reported on linux, it's probably best to just ignore ENODATA for Linux.

So we'll need something like a `isXattrNoData` function. With one version in `node_linux.go` which returns true if the error is ENODATA. And a `node_xattr_bsd.go` which is not build on Linux and all platforms already excluded in `node_xattr.go` (see the `// +build !netbsd` lines).

Reply:
Thanks for the instruction. I don't have much experience about cross-platform developing But I will try it later.

Reply:
I'm using macOS and It also have definition of syscall.ENODATA. So I add a file node_xattr_darwin.go

Reply:
now it removed 😅